### ISSUE TITLE
InvalidArgumentError

### ISSUE THREAD

**Issue by katmayo7** [abhaydoke09/Bilinear-CNN-TensorFlow#23]
Not sure how to clear up this error:
`InvalidArgumentErorr: logits and labels must be the same size: logits_size=[1304,100] labels_size=[32,100]`

This occurs when calling:
`loss = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits=vgg.fc32, labels=target)`

**Comment by katmayo7** [comment]
Not sure what I did. but it seems to have fixed itself.
